src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -139,9 +140,7 @@ private HostAndPort initSentinels(Set<String> sentinels, final String masterName
 
       log.fine("Connecting to Sentinel " + hap);
 
-      Jedis jedis = null;
-      try {
-        jedis = new Jedis(hap.getHost(), hap.getPort());
+      try (Jedis jedis = new Jedis(hap.getHost(), hap.getPort())) {
```

Comment:
Again , it's up to you to decide . Not all fixes/features are to be backported IMHO. It's your call . If you it's crucial to backport , I'll dispose of the try-with-resources . 

If not , let's keep it that way , please :)